android/app/src/main/java/com/on/turip/ui/compose/turip/selection/util/reorderable/ReorderableLazyColumnState.kt

```
@@ -0,0 +1,325 @@
+                        targetItem.offset
+                    }.toFloat()
+
+                withTimeout(1_000L) {
+                    // 1.이동 전 상태 2.이동 후 상태 / 새로운 레이아웃 반영 완료 시점을 동기화하기 위함
+                    layoutInfoFlow.take(2).collect()
+                }
+
+                oldDraggingItemIndex = null
+                predictedDraggingItemOffsetY = null
+            }
+        } catch (e: CancellationException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)